loopdb/swapstate.go

```diff
@@ -127,6 +131,9 @@ func (s SwapState) String() string {
 	case StateInvoiceSettled:
 		return "InvoiceSettled"
 
+	case StateFailIncorrectHtlcAmt:
```

Comment:
nit: wdyt about adding External to the name, so it's clear immediately that this refers to external htlc?

Reply:
Technically it can also happen because of a bug... But yes, maybe better to rename.

Reply:
Tried it, but it also becomes quite a long name causing early line wrapping in some places. I'd say the failure reason is accurate also without External added?

Reply:
Should we decide names based on line length or functionality? :speak_no_evil:

Reply:
Hehe, there is always more detail to add to a name. `StateFailIncorrectExternalLoopInHtlcAmt`

Reply:
I'd say both are factors

Reply:
Generally I'm on the side of more descriptive names (even if it costs us line length), but in this case I'd leave external off because it _could_ happen with an internal htlc.

Reply:
How could this happen with internal htlc?

Reply:
I think only if there is a bug.

Reply:
Leaving short(er) name in place. I do agree with the idea of making it specific, but being complete takes a lot of characters. I think an underlying problem is that we share states between loop in and loop out. There will probably be a split somewhere in the future.